scipy/linalg/_expm_frechet.py

```diff
@@ -67,15 +67,17 @@ def expm_frechet(A, E, method=None, compute_expm=True, check_finite=True):
     Examples
     --------
     >>> import scipy.linalg
-    >>> A = np.random.randn(3, 3)
-    >>> E = np.random.randn(3, 3)
+    >>> rng = np.random.default_rng()
+    >>> A = rng.standard_normal((3, 3))
+    >>> E = rng.standard_normal((3, 3))
     >>> expm_A, expm_frechet_AE = scipy.linalg.expm_frechet(A, E)
     >>> expm_A.shape, expm_frechet_AE.shape
     ((3, 3), (3, 3))
 
     >>> import scipy.linalg
-    >>> A = np.random.randn(3, 3)
-    >>> E = np.random.randn(3, 3)
+    >>> rng = np.random.default_rng()
```

Comment:
```suggestion
```

Do we need to create the default RNG again? It was created just above this block of code. Also looks like we don't need the `scipy.linalg` import above.

Reply:
True for both. I did not want to change too much things around, but I can remove the import too. We are not perfectly consistent along the doc. Sometime we import NumPy, sometimes not, etc.